tests/python_functional/src/syslog_ng_ctl/driver_stats_handler.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+#############################################################################
+# Copyright (c) 2015-2018 Balabit
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 as published
+# by the Free Software Foundation, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+#
+# As an additional exemption you are allowed to compile & link against the
+# OpenSSL libraries as published by the OpenSSL project. See the file
+# COPYING for details.
+#
+#############################################################################
+import src.testcase_parameters.testcase_parameters as tc_parameters
+from src.syslog_ng_ctl.syslog_ng_ctl import SyslogNgCtl
+
+
+class DriverStatsHandler(object):
+    def __init__(self, group_type, driver_name):
+        if group_type == "destination":
+            statement_short_name = "dst"
+        elif group_type == "source":
+            statement_short_name = "src"
+
```

Comment:
Could you raise an Exception here for other `group_type`s?

Reply:
Done